endpoints.json

```diff
@@ -438,9 +438,24 @@
     }
   },
   {
-    "name": "",
+    "name": "searchTextQuery",
+    "method": "GET",
+    "endpoint": "/search/{type}?query={query}",
+    "params": {
+      "type": {
+        "required": false,
+        "optional": true
+      },
+      "query": {
+        "required": false,
+        "optional": true
+      }
+    }
```

Comment:
Both `type` and `query` are required ([ref](http://docs.trakt.apiary.io/#reference/search/text-query/get-text-query-results).